app/components/sidebars/main/channels_list/list/index.js

```diff
@@ -46,7 +46,7 @@ function mapStateToProps(state) {
         sidebarPrefs.grouping,
         sidebarPrefs.sorting,
         sidebarPrefs.unreads_at_top === 'true',
-        sidebarPrefs.favorite_at_top === 'true',
+        sidebarPrefs.favorite_at_top === 'true' && favoriteChannelIds.length,
```

Comment:
Should we submit the same fix for the webapp?

Reply:
@saturninoabril I don't know if this is also happening on the webapp, the ticket is about RN

Reply:
I understand, maybe something to watch out on the webapp since it has similar implementation there.